### ISSUE TITLE
SDK Download

### ISSUE THREAD

**Issue by alesimoes** [ant-thomas/zsgx1hacks#92]
Hi, where can I download the goke SDK?

**Comment by paus56** [comment]
http://en.pudn.com/Download/item/id/3294300.html

**Comment by dst0815** [comment]
You are right, deorder. The file doesnt contain any kernel code, just informations in mandarin, mainly for RTOS, not linux. I uploaded the file to my google drive, its at https://drive.google.com/uc?id=1gMJqD-cdlKxbfHqc8gLws2wpt1hfNs-p&export=download . Also, if anyone needs the toolchain for other platforms, they are at https://launchpad.net/gcc-arm-embedded/4.9/4.9-2015-q1-update . Best regards, John

**Comment by deorder** [comment]
Thank you. There is a SDK that is reported to work with the Goke as well. Have not tested it yet. You can find the download at the bottom of: https://github.com/Filipowicz251/mijia-1080P-hacks/wiki

**Comment by paus56** [comment]
**[GK710X Linux SDK](https://github.com/ant-thomas/zsgx1hacks/issues/124)**
